src/components/sign-up-second/index.tsx

```diff
@@ -45,146 +40,146 @@ function SignUpSecondForm({ role, onNext }: IProps): JSX.Element {
+            type="text"
+          />
+        </FormControl>
+        {errors?.lastName && (
+          <ErrorMessage variant="caption">{errors.lastName?.message}</ErrorMessage>
+        )}
+      </InputWrapper>
+      <InputWrapper>
+        <FormControl sx={{ width: '100%' }} variant="filled">
+          <InputLabel htmlFor="email">{translate('signUpSecondForm.placeholderEmail')}</InputLabel>
+          <FilledInput {...register('email')} id="email" error={!!errors.email} type="email" />
+        </FormControl>
+        {errors?.email && <ErrorMessage variant="caption">{errors.email?.message}</ErrorMessage>}
+      </InputWrapper>
+      <InputWrapper>
+        <FormControl sx={{ width: '100%' }} variant="filled">
+          <Controller
+            name="phoneNumber"
+            control={control}
+            render={({ field, fieldState }): JSX.Element => (
+              <MuiTelInput
+                {...field}
+                onChange={(value: string): void => {
+                  field.onChange(value);
+                  if (value.slice(2, 3) === '0') {
+                    field.onChange('');
+                    setError('phoneNumber', {
+                      type: 'manual',
+                      message: `${translate('signUpSecondForm.phoneInvalid')}`,
+                    });
+                    return;
+                  }
+                  clearErrors('phoneNumber');
+                }}
+                variant="filled"
+                defaultCountry="US"
+                focusOnSelectCountry
+                disableFormatting
+                forceCallingCode
+                onlyCountries={['US', 'CA']}
+                error={fieldState.invalid}
+                inputRef={Input}
+                label={translate('signUpSecondForm.placeholderPhone')}
+              />
+            )}
+          />
+        </FormControl>
+        {errors?.phoneNumber && (
+          <ErrorMessage variant="caption">{errors.phoneNumber?.message}</ErrorMessage>
+        )}
+      </InputWrapper>
+      <InputWrapper>
+        <FormControl sx={{ width: '100%', height: 48 }} variant="filled">
+          <InputLabel htmlFor="dateOfBirth">
+            {translate('signUpSecondForm.placeholderBirthDate')}
+          </InputLabel>
+          <Controller
+            control={control}
+            name="dateOfBirth"
+            render={({ field }): JSX.Element => (
+              <DatePicker
+                onChange={(date: Date): void => field.onChange(date)}
+                selected={field.value}
+                customInput={<FilledInput fullWidth error={!!errors.dateOfBirth} />}
+                maxDate={minBirthDate}
+                dateFormat={USER_DATE_BIRTH_FORMAT}
+              />
+            )}
+          />
+        </FormControl>
+        {errors?.dateOfBirth && (
+          <ErrorMessage variant="caption">{errors.dateOfBirth?.message}</ErrorMessage>
+        )}
+      </InputWrapper>
+      {role === 'caregiver' && (
```

Comment:
1.1.5 https://www.notion.so/Git-rules-849f8ca8b3db4ded926d762c6fff8798

Reply:
fixed. add user roles to root constants